tests/FlatBuffers.Test.Swift/Tests/FlatBuffers.Test.SwiftTests/monster_test_generated.swift

```diff
@@ -5,209 +5,400 @@ import FlatBuffers
 public enum MyGame {
 public enum Example {
 
-/// Composite components of Monster color.
-public enum Color: UInt8, Enum { 
-	public typealias T = UInt8
-	public static var byteSize: Int { return MemoryLayout<UInt8>.size }
-	public var value: UInt8 { return self.rawValue }
-	case red = 1
-	/// \brief color Green
-	/// Green is bit_flag with value (1u << 1)
-	case green = 2
-	/// \brief color Blue (1u << 3)
-	case blue = 8
-
-
-	public static var max: Color { return .blue }
-	public static var min: Color { return .red }
-}
-
-public enum Race: Int8, Enum { 
-	public typealias T = Int8
-	public static var byteSize: Int { return MemoryLayout<Int8>.size }
-	public var value: Int8 { return self.rawValue }
-	case none = -1
-	case human = 0
-	case dwarf = 1
-	case elf = 2
-
-
-	public static var max: Race { return .elf }
-	public static var min: Race { return .none }
-}
-
-public enum Any_: UInt8, Enum { 
-	public typealias T = UInt8
-	public static var byteSize: Int { return MemoryLayout<UInt8>.size }
-	public var value: UInt8 { return self.rawValue }
-	case none = 0
-	case monster = 1
-	case testsimpletablewithenum = 2
-	case mygame_example2_monster = 3
-
-
-	public static var max: Any_ { return .mygame_example2_monster }
-	public static var min: Any_ { return .none }
-}
-
-public enum AnyUniqueAliases: UInt8, Enum { 
-	public typealias T = UInt8
-	public static var byteSize: Int { return MemoryLayout<UInt8>.size }
-	public var value: UInt8 { return self.rawValue }
-	case none = 0
-	case m = 1
-	case ts = 2
-	case m2 = 3
+    ///  Composite components of Monster color.
+    public enum Color: UInt8, Enum { 
```

Comment:
still indented?

Reply:
i thought you meant just for the namespaces, didn't think of the enums, structs, and classes